### PR TITLE
Copter: Updates to vibration analsys for mechanical issues.

### PR DESCRIPTION
**Pull request by Allister779**
There was a request made to add this information from a forum post.

https://discuss.ardupilot.org/t/imus-reporting-vibration-on-different-axis/82415/2?u=allister

### CODE REVIEW

**Review by rmackay9 — APPROVED** [review]
seems like good advice to me..

File: common/source/docs/common-measuring-vibration.rst [hunk 1]
```diff
@@ -72,6 +72,16 @@ the accelerometer readings like this:
    square root (for x, y and z).  These final three values are what
    appear in the VIBE msg's VibeX, Y and Z fields.
 
+When looking at vibrations in a log the first thing to look at is the clipping. If the clipping is 0 that’s good. That means the vibrations that are being detected aren’t overwhelming the IMU.
+
+When troubleshooting the vibration, consider the axis of the vibration as a staring point to find the problem:
+
+    If X AND Y are both high, then you may have an issue with a motor bearing or prop balance. Or you may need more/better overall vibration damping for your FC.
+    If X OR Y is high then you may have an issue with your FC mounting. Maybe a wire or is bouncing on the FC or restraining it. Or you maybe your vibration damping works better one way than the other.
```
Comment (hendjoshsr71) [inline common/source/docs/common-measuring-vibration.rst:80]:
```suggestion
    If X OR Y is high then you may have an issue with your FC mounting. Maybe a wire is bouncing on the FC or restraining it. Or maybe your vibration damping works better one way than the other.
```

File: common/source/docs/common-measuring-vibration.rst [hunk 2]
```diff
@@ -72,6 +72,16 @@ the accelerometer readings like this:
...
+    If you have a Z vibration then you may have a track issue with propeller (bent blade) or vertical play in a motor.
+
+Also consider that some flight conditions/airframes will have different natural vibrations. If the vibrations look good in a hover,  but they goes up with speed, perhaps there is an aerodynamic issue on the airframe, or the wind,
```
Comment (hendjoshsr71) [inline common/source/docs/common-measuring-vibration.rst:83]:
```suggestion
Also consider that some flight conditions/airframes will have different natural vibrations. If the vibrations look good in a hover, but they goes up with speed, perhaps there is an aerodynamic issue on the airframe, or the wind,
```

**Review by hendjoshsr71 — COMMENTED** [review]
Nice! Thanks!

Don't feel like you must make the changes.  Myself or Henry can fix it up if you agree with them.   

And we can squash the commits if you need us to.

File: common/source/docs/common-measuring-vibration.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-measuring-vibration.rst:83]: Josh...tweaked it a bit

File: common/source/docs/common-measuring-vibration.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-measuring-vibration.rst:80]: tweaked here too

Comment (hendjoshsr71) [reply common/source/docs/common-measuring-vibration.rst:80]: looks good  to me ! Thanks!

File: common/source/docs/common-measuring-vibration.rst (hunk 2 above)
Comment (hendjoshsr71) [reply common/source/docs/common-measuring-vibration.rst:83]: looks good  to me ! Thanks!

File: common/source/docs/common-measuring-vibration.rst (hunk 1 above)
Comment (Allister779) [reply common/source/docs/common-measuring-vibration.rst:80]: I'm good with it.  Thanks.

File: common/source/docs/common-measuring-vibration.rst (hunk 2 above)
Comment (Allister779) [reply common/source/docs/common-measuring-vibration.rst:83]: Thank!  This is my first PR so I'm not sure if there's anything I need to do now? (or have access to do?)

**Comment by hendjoshsr71** [comment]
Yes @Allister779  thank you so much for the addition!

**Pull request merged**